aws/resource_aws_lambda_alias.go

```diff
@@ -117,6 +120,7 @@ func resourceAwsLambdaAliasRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("function_version", aliasConfiguration.FunctionVersion)
 	d.Set("name", aliasConfiguration.Name)
 	d.Set("arn", aliasConfiguration.AliasArn)
+	d.SetId(*aliasConfiguration.AliasArn)
```

Comment:
Typically `d.SetId()` should only be called during `Create` and `Importer` `State` functions, however this is understandably a little more nuanced here since if we followed that rule here we would need the entire alias ARN passed into `terraform import` or to make an additional API call in the import function which is less than ideal. I'm okay with merging this as-is for now to get this functionality in. 👍